src/Compilers/CSharp/Portable/Symbols/SubstitutedNamedTypeSymbol.cs

```diff
@@ -159,6 +159,9 @@ internal override ImmutableArray<NamedTypeSymbol> GetInterfacesToEmit()
             throw ExceptionUtilities.Unreachable;
         }
 
+        internal override bool GetUnificationUseSiteDiagnosticRecursive(ref DiagnosticInfo result, Symbol owner, ref HashSet<TypeSymbol> checkedTypes)
```

Comment:
>GetUnificationUseSiteDiagnosticRecursive [](http://example.com/codeflow?start=31&length=40)

If this implementation is specific to `SubstitutedNestedTypeSymbol`, perhaps the implementation should be there with this `override` being `abstract`.